### PR TITLE
Refactor scripts

### PR DESCRIPTION
**Pull request by nygrenh**

### CODE REVIEW

File: scripts/bash_install.sh [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+#!/bin/bash
 
-cd $( dirname "${BASH_SOURCE[0]}" )
+cd "$( dirname "${BASH_SOURCE[0]}" )"
 TMCDIR=$(pwd)
 
-echo -e '\n### Added by TestMyCode Commandline client' >> $HOME/.bashrc
-echo "export PATH=\"$TMCDIR:\$PATH\"" >> $HOME/.bashrc
-source $HOME/.bashrc
+echo -e '\n### Added by TestMyCode Commandline client' >> "$HOME/.bashrc"
```
Comment (jamox) [inline scripts/bash_install.sh]: Do we need/want this (does rvm, rbenv and others print this)

Oh and what about other shells (zsh, fish etc)

File: scripts/bash_install.sh [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+echo "export PATH=\"$TMCDIR:\$PATH\"" >> "$HOME/.bashrc"
+source "$HOME/.bashrc"
 
 echo Install finished. Type "tmc" to get started.
```
Comment (jamox) [inline scripts/bash_install.sh]: Should this be nice green?

File: scripts/shutdown.sh [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
 DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
-CONFIGPATH=$DIR
-CONFIGPATH+="/config"
+PIDFILE="$DIR/pidfile.pid"
+PID="$(cat "$PIDFILE")"
 
-kill -9 `cat $CONFIGPATH`
-echo "" > $CONFIGPATH
+kill "$PID"
+sleep 2
```
Comment (jamox) [inline scripts/shutdown.sh]: is 2 sec enough for possible final messages/ongoing transmissions? -- submit on slow internet.

maybe put the killer to background with 10-30sec delay?

File: scripts/bash_install.sh (hunk 1 above)
Comment (zinkko) [inline scripts/bash_install.sh]: The heroku toolbelt adds a comment like this.
Other shells probably will need their own install scripts, that's why this script is named bash_install

Comment (jamox) [inline scripts/bash_install.sh]: Ack

**Pull request merged**